src/python/Publisher/TaskPublishRucio.py

```diff
@@ -15,7 +15,7 @@
 from TaskWorker.WorkerExceptions import CannotMigrateException
 from TaskWorker.WorkerUtilities import getCrabserver
 
-from PublisherUtils import setupLogging, prepareDummySummary, saveSummaryJson, \
+from Publisher.PublisherUtils import setupLogging, prepareDummySummary, saveSummaryJson, \
```

Comment:
I never understood those things fully. Why you need to add `Publisher.` prefix here but not in the following line ?

Reply:
comparing with file above (I was reading bottom up apparently), it seems an oversight

Reply:
Nice catch!
Thanks